### PR TITLE
Closes #49

### LINKED ISSUE #49: Create a dataset loader for Scielo
From https://sites.google.com/view/felipe-soares/datasets#h.p_92uSCyAjWSRB

### PR DESCRIPTION
**Pull request by WojciechKusa**
closes #49

### NOTE 1: 
bigbio_t2t schema doesn't allow more than two texts in text-to-text schema.  en-pt-es translation is not implemented using the bigbio schema

### CODE REVIEW

File: biodatasets/scielo/scielo.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+
+class ScieloDataset(datasets.GeneratorBasedBuilder):
+    """Parallel corpus of full-text articles in Portuguese, English and Spanish from SciELO."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    # NOTE: bigbio_t2t schema doesn't allow only for more than two texts in text-to-text schema.
+    #  en-pt-es translation is not implemented using the bigbio schema
```
Comment (galtay) [inline biodatasets/scielo/scielo.py:65]: maybe not ideal, but you could define a `source` schema and config that included all languages 
```
BigBioConfig(
            name="scielo_source",
            version=SOURCE_VERSION,
            description="scielo all languages",
            schema="source",
            subset_id="scielo",
        ),
```

Comment (WojciechKusa) [reply biodatasets/scielo/scielo.py:65]: great idea, I will fix it!

**Review by galtay — CHANGES_REQUESTED** [review]
changing review to "request changes" for easier tracking in PR list

**Comment by hakunanatasha** [comment]
@galtay Assigned for your review since you have some familiarity with the PR; please ping me if you need help

**Comment by galtay** [comment]
@WojciechKusa  is this still on your radar?

**Comment by WojciechKusa** [comment]
Yes, I am still working on it

**Comment by hakunanatasha** [comment]
@WojciechKusa will unit-tests still fail if you call them via the `--subset_id` flag?

**Comment by WojciechKusa** [comment]
I changed the `subset_id` params for the `BigBioConfig`. Now unit tests work fine with: `python -m tests.test_bigbio biodatasets/scielo/scielo.py --subset_id scielo_en-es`

`scielo_en-pt-es_source` handles three languages in source schema

You can take a look one more time @galtay @hakunanatasha

File: biodatasets/scielo/scielo.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+            description="English-Portuguese-Spanish",
+            schema="source",
+            subset_id="scielo_en-pt-es",
+        ),
+        BigBioConfig(
+            name="scielo_en-es_bigbio_t2t",
+            version=BIGBIO_VERSION,
+            description="scielo BigBio schema English-Spanish",
+            schema="bigbio_t2t",
+            subset_id="scielo",
```
Comment (galtay) [inline biodatasets/scielo/scielo.py:94]:
```suggestion
            subset_id="scielo_en-es",
```

File: biodatasets/scielo/scielo.py [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+        ),
+        BigBioConfig(
+            name="scielo_en-pt_bigbio_t2t",
+            version=BIGBIO_VERSION,
+            description="scielo BigBio schema English-Portuguese",
+            schema="bigbio_t2t",
+            subset_id="scielo",
```
Comment (galtay) [inline biodatasets/scielo/scielo.py:101]:
```suggestion
            subset_id="scielo_en_pt",
```

**Review by galtay — CHANGES_REQUESTED** [review]

**Review by galtay — APPROVED** [review]

**Comment by galtay** [comment]
🎉

**Pull request merged**